docs/ref/schema-editor.txt

```diff
@@ -184,3 +184,9 @@ accessed.
 
 This is useful when doing data migrations for :ref:`migrations with multiple
 databases <data-migrations-and-multiple-databases>`.
+
+Template strings
+----------------
+
+The ``SchemaEditor`` contains a number of template strings for creating and
```

Comment:
I'm trying to make sense of this sentence. It feels to me like a half-finished paragraph. :-/